src/cwv/handler.js

```diff
@@ -44,7 +45,69 @@ export async function CWVRunner(auditUrl, context, site) {
   };
 }
 
+export async function convertToOppty(auditUrl, auditData, context) {
+  const {
+    dataAccess,
+    log,
+  } = context;
+
+  const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  let opportunity = opportunities.find((oppty) => oppty.getType() === AUDIT_TYPE);
+
+  if (!opportunity) {
+    const opportunityData = {
+      siteId: auditData.siteId,
+      auditId: auditData.id,
+      runbook: 'https://adobe.sharepoint.com/sites/aemsites-engineering/Shared%20Documents/3%20-%20Experience%20Success/SpaceCat/Runbooks/Experience_Success_Studio_CWV_CLS_Runbook.docx?web=1',
+      type: AUDIT_TYPE,
+      origin: 'AUTOMATION',
+      title: 'Core Web Vitals',
+      description: 'Core Web Vitals are key metrics Google uses to evaluate website performance, impacting SEO rankings by measuring user experience.',
+      guidance: {
+        steps: [
+          'Analyze CWV data using RUM and PageSpeed Insights to identify performance bottlenecks.',
+          'Optimize CWV metrics (CLS, INP, LCP) by addressing common issues such as slow server response times, unoptimized assets, excessive JavaScript, and layout instability.',
+          'Test the implemented changes with tools like Chrome DevTools or PageSpeed Insights to verify improvements.',
+          'Monitor performance over time to ensure consistent CWV scores across devices.',
+        ],
+      },
+      tags: [
+        'SEO',
+        'Performance Optimization',
+        'User Experience',
+        'Core Web Vitals',
+        'Organic Traffic',
+        'Web Performance',
+      ],
+    };
+    opportunity = await dataAccess.Opportunity.create(opportunityData);
```

Comment:
add error handling as done here:
https://github.com/adobe/spacecat-audit-worker/pull/475/files#diff-212dd58956c947938959c789cc3912af8f1b884de5543486166247e12f8ef215R188

Reply:
@habansal, I don't think we should swallow the error there. But I added a try/catch statement with error logging. The error should be handled one-level above, not here

Reply:
@habansal just saw this comment after I merged the PR. I will update the example in the README in a separate PR

Reply:
see: https://github.com/adobe/spacecat-audit-worker/pull/506